### PR TITLE
Upgrade ANF SDK version

### PR DESCRIPTION
**Pull request by AdelGhabboun**
1. Upgraded ANF SDK version to 1.13.0
2. Upgraded .NET version to .NET 5.0
3. Added logic to read password from Console.

### CODE REVIEW

**Review by hildurhodd — APPROVED** [review]
LGTM

File: src/anf-dotnetcore-dual-protocol-sdk-sample/Program.cs [hunk 1]
```diff
@@ -66,6 +67,16 @@ static void Main(string[] args)
 
         private static async Task CreateANFAsync()
         {
+            Console.WriteLine("Please type Active Directory's user password that will domain join ANF's SMB server and press [ENTER]:");
+
+            string DomainJoinUserPassword = Utils.GetConsolePassword();
+
+            // Basic validation
+            if (string.IsNullOrWhiteSpace(DomainJoinUserPassword))
```
Comment (paulomarquesc) [inline src/anf-dotnetcore-dual-protocol-sdk-sample/Program.cs]: >DomainJoinUserPassword [](http://example.com/codeflow?start=42&length=22)

Camel Case #Resolved

File: src/anf-dotnetcore-dual-protocol-sdk-sample/Program.cs [hunk 2]
```diff
@@ -98,7 +109,7 @@ private static async Task CreateANFAsync()
                 location,
                 anfAccountName,
                 domainJoinUsername,
-                domainJoinPassword,
+                DomainJoinUserPassword,
```
Comment (paulomarquesc) [inline src/anf-dotnetcore-dual-protocol-sdk-sample/Program.cs]: >DomainJoinUserPassword [](http://example.com/codeflow?start=16&length=22)

Camel Case

**Review by paulomarquesc — CHANGES_REQUESTED** [review]
:clock1:

File: src/anf-dotnetcore-dual-protocol-sdk-sample/Program.cs [hunk 3]
```diff
@@ -66,6 +67,16 @@ static void Main(string[] args)
 
         private static async Task CreateANFAsync()
         {
+            Console.WriteLine("Please type Active Directory's user password that will domain join ANF's SMB server and press [ENTER]:");
+
+            string DomainJoinUserPassword = Utils.GetConsolePassword();
```
Comment (paulomarquesc) [inline src/anf-dotnetcore-dual-protocol-sdk-sample/Program.cs]: >DomainJoinUserPassword [](http://example.com/codeflow?start=19&length=22)

Camel Case #Resolved

**Review by paulomarquesc — APPROVED** [review]
:shipit:

**Pull request merged**